_episodes/08-bonus-episode.md

```diff
@@ -84,38 +83,38 @@ It is possible to publish singularity images with GitHub packages. To do so, one
 name: Singularity Build Deploy
 
 on:
-  pull_request:
-  push:
-    branches: master
+  pull_request:
+  push:
+    branches: master
 
 jobs:
-  build-test-container:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      packages: write
-
-    container:
-        image: quay.io/singularity/singularity:v3.8.1
-        options: --privileged
-
-    name: Build Container
-    steps:
-
-      - name: Check out code for the container builds
-        uses: actions/checkout@v2
-
-      - name: Build Container
-        run: |
-         sudo -E singularity build container.sif Singularity
-
-      - name: Login and Deploy Container
-        run: |
-           echo ${{ secrets.GITHUB_TOKEN }} | singularity remote login -u ${{ secrets.GHCR_USERNAME }} --password-stdin oras://ghcr.io
-           singularity push container.sif oras://ghcr.io/${GITHUB_REPOSITORY}:${tag}
+  build-test-container:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    container:
+        image: quay.io/singularity/singularity:v3.8.1
```

Comment:
I'm not really sure what's going on here with the whitespace, isn't this mixing tabs and spaces here?